### PR TITLE
Migrating from infra to single-cell-infra

### PR DESCRIPTION
**Pull request by Bento007**
- Renaming dcp to corpora
- Renaming browser to data-portal
- tearing down old dev and staging infra
- deploying new dev and staging infra using single-cell-infra

Depends on https://github.com/chanzuckerberg/single-cell-infra/pull/75

### CODE REVIEW

File: browser/backend/code/common/authorizer.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
 USERINFO_ENDPOINT = "https://czi-single-cell.auth0.com/userinfo"
 
 AUTH0_DOMAIN = "czi-single-cell.auth0.com"
-API_AUDIENCE = f"https://browser-api.{os.environ['DEPLOYMENT_STAGE']}.single-cell.czi.technology"
+API_AUDIENCE = f"https://api.corpora.{os.environ['DEPLOYMENT_STAGE']}.single-cell.czi.technology"
```
Comment (calvinnhieu) [inline browser/backend/code/common/authorizer.py:11]: Do we have plans to use the domain `corpora.cziscience.com`?

Comment (maniarathi) [reply browser/backend/code/common/authorizer.py:11]: Yep, it should be the one that Calvin mentioned.

File: browser/backend/code/config/db_config.py [hunk 2]
```diff
@@ -3,4 +3,4 @@
 
 class BrowserDbConfig(Config):
```
Comment (maniarathi) [inline browser/backend/code/config/db_config.py:4]: Rename to DataPortalDbConfig?

File: browser/backend/code/common/authorizer.py (hunk 1 above)
Comment (Bento007) [reply browser/backend/code/common/authorizer.py:11]: What is the URL for the site, and what is the URL for the api?

Comment (maniarathi) [reply browser/backend/code/common/authorizer.py:11]: corpora.cziscience.com
api.corpora.cziscience.com

Comment (Bento007) [reply browser/backend/code/common/authorizer.py:11]: These domains are for productions only right? It looks like cellxgene is using cellxgene.staging.single-cell.czi.technology for is development stage. Do we want to follow that same naming scheme as cellxgene for our development environements?

Comment (maniarathi) [reply browser/backend/code/common/authorizer.py:11]: You'd tag on the `staging` or `dev` environment name appropriately.

I spoke with Bruce and Madison about the cellxgene URLs and they largely used them due to convenience and bc they were free. We aren't tied to matching the same naming scheme as cellxgene so let's keep ours simple and only use the cziscience.com domain unless there's a strong reason not to.

**Pull request merged**